### PR TITLE
Performance page: add intro with performance design

### PR DESCRIPTION
**Pull request by arl**
Add intro on performance page, explaining baker aimed at performance from the start

### CODE REVIEW

File: content/en/docs/Performance/_index.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ description: >
   Baker performance test cases
 ---
 
+Baker has been designed with high performance in mind. Baker core, the part of the code base
+which distributes records among components and ties them together, is very high-quality Go code.
+Records are never copied, and a particular attention has been given to reduce the number of
+memory allocations as much as possible, so as to keep the garbage collector cost to a minimum.
+
+Baker is also battle-tested, since 2016 NextRoll has been running hundreds if not thousands
+of Baker pipelines, daily.
```
Comment (tommyblue) [inline content/en/docs/Performance/_index.md:15]:
```suggestion
of Baker pipelines, daily, processing PetaBytes of lines.
```

File: content/en/docs/Performance/_index.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ description: >
   Baker performance test cases
 ---
 
+Baker has been designed with high performance in mind. Baker core, the part of the code base
+which distributes records among components and ties them together, is very high-quality Go code.
```
Comment (tommyblue) [inline content/en/docs/Performance/_index.md:10]:
```suggestion
which distributes records among components and ties them together, is very high-quality Go code.  
```

Comment (tommyblue) [reply content/en/docs/Performance/_index.md:10]: double space to add a newline

**Review by tommyblue — APPROVED** [review]
lgtm, just added 2 minor comments

File: content/en/docs/Performance/_index.md (hunk 2 above)
Comment (arl) [reply content/en/docs/Performance/_index.md:10]: i know but I don't want a new line here :-)

Comment (tommyblue) [reply content/en/docs/Performance/_index.md:10]: ok

**Pull request merged**